instadm.py

```diff
@@ -126,6 +126,18 @@ def typeMessage(self, user, message):
             self.__random_sleep__()
 
         if self.__wait_for_element__(self.selectors['send'], "xpath"):
+            element = self.__get_element__('rh7Wz', 'CLASS')
+            if element is not None:
+                self.driver.execute_script("""
+                                    var element = arguments[0];
+                                    element.parentNode.removeChild(element);
+                                    """, element)
+            element = self.__get_element__('vohlx', 'CLASS')
+            if element is not None:
+                self.driver.execute_script("""
+                                    var element = arguments[0];
+                                    element.parentNode.removeChild(element);
+                                    """, element)
```

Comment:
Could you move this code into a function? :)

Reply:
Done :)